compiler/rustc_trait_selection/src/solve/alias_relate.rs

```diff
@@ -81,9 +83,7 @@ impl<'tcx> EvalCtxt<'_, 'tcx> {
                     // As this is incomplete, we must not do so during coherence.
                     match self.solver_mode() {
```

Comment:
this comment is wrong, please also link to https://github.com/rust-lang/trait-system-refactor-initiative/issues/25